landing/views.py

```diff
@@ -4,5 +4,5 @@
 # from .forms import MessageForm
 
 
-def indexView(request):
+def indexview(request):
```

Comment:
You don't need to add the views.py file for the current demo

Reply:
Done